src/Sentry/PlatformAbstractions/FrameworkInfo.NetFx.cs

```diff
@@ -85,8 +88,19 @@ public static partial class FrameworkInfo
         /// <returns>Enumeration of installations</returns>
         public static IEnumerable<FrameworkInstallation> GetInstallations()
         {
-            using var ndpKey = RegistryKey.OpenRemoteBaseKey(RegistryHive.LocalMachine, string.Empty)
-                .OpenSubKey(@"SOFTWARE\Microsoft\NET Framework Setup\NDP\");
+            try
+            {
+                using var ndpKey = Registry.LocalMachine.OpenSubKey(NetFxNdpRegistryKey, false);
+                return GetInstallationsFromRegistryKey(ndpKey).ToArray();
+            }
+            catch
```

Comment:
Personally not a fan of just catch and hiding things. Can we either pass a logger and log the issue or let it bubble and handle this at the caller?